packages/cli/src/commands/set-admin.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import setAdmin from '../scripts/set-admin'
+import runWithTruffle from '../utils/runWithTruffle'
+import { fromContractFullName } from '../utils/naming'
+import _ from 'lodash'
+
+const name = 'set-admin'
+const signature = `${name} [alias-or-address] [new-admin-address]`
+const description = 'change upgradeability admin of a contract instance. Provide the [alias] or [package]/[alias] you added your contract with, or its [address]. Note that if you transfer to an incorrect address, you may irreversibly lose control over upgrading your contract.'
```

Comment:
I would make clear that in case of an alias we are going to set the admin of all the instances for that alias